examples/load_balancer/load_balancer.c

```diff
@@ -143,19 +141,43 @@ usage(const char *progname) {
  */
 static int
 parse_app_args(int argc, char *argv[], const char *progname) {
-        int c;
+        int c, ret;
+        char *client_args, *server_args;
+        char client_ip[32], server_ip[32];
 
         lb->cfg_filename = NULL;
-        lb->client_iface_name = NULL;
-        lb->server_iface_name = NULL;
+        lb->client_port = RTE_MAX_ETHPORTS+1;
```

Comment:
Setting to the max value + 1 seems a bit strange to me - I would set to 0 or -1 and add a check to see if the set value falls between the valid port range.

Reply:
I originally intended to set the port to 0 or -1, but the port's data type is an unsigned integer and 0 is a valid port ID. In my mind, this was the next best option to check whether the ID had been validated (though I now realize that I could do max rather than max+1). Let me know what you think

Reply:
Ah I see your reasoning. I was conflating this with normal ports. Just MAX sounds good to me.